addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/EnvFileScanRule.java

```diff
@@ -19,13 +19,26 @@
  */
 package org.zaproxy.zap.extension.ascanrulesBeta;
 
+import java.util.regex.Pattern;
 import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.network.HttpResponseHeader;
 import org.zaproxy.addon.commonlib.AbstractAppFilePlugin;
 
 public class EnvFileScanRule extends AbstractAppFilePlugin {
-
+    /**
+     * Environment files come in many flavors but mostly they are KEY=VALUE formatted Here's the
+     * trick, NGINX returns them as binary/octet-stream content-type Apache just returns the text
+     * with no content-type Just looking for content returned with status code 200 and '#' and '='
+     * is a FP nightmare Because that's basically ALL html files
+     */
```

Comment:
Should be a plain comment, and probably better move closer to what's referring to.

Reply:
moved this into a better place as a normal comment